### PR TITLE
ab: edit text in select vars

### LINKED ISSUE #398: Experiment setup: Text edits on the 'Select variables' page
The text instructions on this page can read:
Select up to three experimental variables, in addition to strains and treatments, that you wish to define for your treatment protocols.

### PR DESCRIPTION
**Pull request by annagav**
Fix #398, #394

### CODE REVIEW

File: instructor/templates/instructor/assignment_sidebar.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
             <a class='scb_s_assignment_sidebar_link'
                href="{% if new %}{% url "common_assignment_setup" %}
                {% else %}{% url "common_assignment_modify" %}{% endif %}">
-                 Assignment setup
+                 Assignment
```
Comment (noisecapella) [inline instructor/templates/instructor/assignment_sidebar.html]: It looks like #394 was to change the heading and leave the subheading alone, but this changes the subheading and leaves the heading alone

**Comment by annagav** [comment]
Fixed.

**Comment by noisecapella** [comment]
Can you rebase please?

**Comment by annagav** [comment]
Done

**Comment by noisecapella** [comment]
:+1:

**Pull request merged**